### DISCUSSION TITLE
How to minimize the nav bar folder (group of pages) by default?

### DISCUSSION THREAD

**Discussion by bestickley** [shuding/nextra#2168] [Q&A] [answered]
Hi, how can I minimize the nav bar folder (group of pages) by default?

**Comment by devjiwonchoi** [comment, marked as answer]
In your `theme.config.tsx`, set `sidebar.defaultMenuCollapseLevel` to 1.

> By default, the sidebar menu is collapsed at level 2. You can change it by setting sidebar.defaultMenuCollapseLevel to a different number. For example, when set to 1, every folder will be collapsed by default and when set to Infinity, all nested folders will be expanded by default.

```ts
// theme.config.tsx

import { DocsThemeConfig } from 'nextra-theme-docs'

const config: DocsThemeConfig = {
  // ...
  sidebar: {
    defaultMenuCollapseLevel: 1, // here
  },
}

export default config
```

### Reference
- [Sidebar](https://nextra.site/docs/docs-theme/theme-configuration#sidebar)
